desk/app/grouper.hoon

```diff
@@ -135,11 +135,28 @@
       %fact
     =+  !<(=bite:reel q.cage.sign)
     ?>  ?=([%bite-2 *] bite)
+    =>
+      |%
+      ++  lure-log
+        |=  [=volume:logs event=@t =echo:logs]
+        %^  tell:log  volume
+          echo
+        :~  'event'^s+event
+            'flow'^s+'lure'
+            'lure-id'^s+token.bite
+            'lure-joiner'^s+(scot %p joiner.bite)
+        ==
+      --
     :_  this
     =;  caz=(list card)
       ?~  inviter=(~(get by fields.metadata.bite) 'inviter')
-        ~[(tell:log `token.bite %crit 'inviter field missing' ~)]
-      ?.  =((slav %p u.inviter) our.bowl)  ~
+        :_  ~
+        %^  lure-log  %crit  'Inviter Missing'
+        ~['inviter field missing in lure bite']
+      ?.  =((slav %p u.inviter) our.bowl)
+        :_  ~
+        %^  lure-log  %crit  'Foreign Inviter'
```

Comment:
Can we change the `eventName`'s for all the failures to be _Sending DM Invite Failed_ or _Sending Group Invite_ failed?

That way it'll be super clear there's a problem + easier to aggregate and we can look to the message property for additional context